.circleci/config.yml

```diff
@@ -1118,6 +1172,22 @@ jobs:
           name: Validate source maps
           command: yarn validate-source-maps
 
+  validate-source-maps-desktop:
```

Comment:
Definitely beyond the scope of this PR, but for any build type specific jobs with a very similar structure, we could use CircleCI commands to define the steps once with a `buildType` parameter and avoid some of this duplication.